### DISCUSSION TITLE
Fahrbereiche

### DISCUSSION THREAD

**Discussion by Jonas2859** [Mogli12/KeyboardSteer#811] [Ideas]
Hallo,

Erst mal danke für VCA, es hat den LS im 19er schon einiges realistischer gemacht und jetzt im 22er ebenfalls.

Wäre es auch möglich Fahrbereiche so wie es in den neueren Case und Steyr Traktoren der fall ist hinzuzufügen?
Also ich stelle es mir so vor dass man z.B. die 3 Fahrbereiche hat und wie in der realität für jeden Fahrbereich die Endgeschwindigkeit separat einstellen kann und die Bereiche dann mit 2 Tasten durchschaltet.

Ich würde mich freuen wenn dir die Idee gefällt und du es verbaust.
Bei fragen gerne per Discord melden: Jonas359#3746

Liebe Grüße 
Jonas

**Comment by RysingTV** [comment]
Hi, ich habe für den Tempomat verschiedene Geschwindigkeiten also was würde mir jetzt dein Vorschlag bringen? 

Ich frage mal Interesse halber da ich für mich versuche einen son zu sehen.

**Reply by Jonas2859** [reply]
Wenn man den Tempomat aktiviert dann hält er die Geschwindigkeit, das ist hier aber nicht der fall es wird nur die Endgeschwindigkeit heruntergesetzt. Man kann jedoch bis zu dieser Endgeschwindigkeit in dem jeweiligen Fahrbereich immer noch selber die Geschwindigkeit mit dem Gaspedal regeln.

**Comment by Mogli12** [comment]
VCA macht zwar ein paar Anpassungen rund um das Getriebe. Aber das Getriebe selber wird von VCA nicht mehr verändert.
